### DISCUSSION TITLE
Size of buttons!

### DISCUSSION THREAD

**Discussion by suneworld** [punxaphil/custom-sonos-card#104] [Q&A] [answered]
Hi!
Thank you for an amazing card!

I was wondering if there is any way to make the media player buttons larger? 

Kindest regards from Norway

**Comment by punxaphil** [comment]
Yes, you need to look into the styles configuration and use some CSS. Take a look at the documentation.

**Comment by richardwonka** [comment]
Do you have a link to the relevant bit of the documentation? I haven’t found it. 

I think having a large, centred play/pause button and maybe forward and shuffle would be a very useful default instead of the very small buttons in the bottom panel, hidden among things that likely see less use.

**Comment by punxaphil** [comment]
These are the instructions:

**CSS Styling**
For maximum control of look and feel, define your style with CSS under styles.

Many elements in the card can be styled using this, but not all. Using your web browser's developer console, inspect the element and check the CSS. If the CSS contains --sonos-card-style-name: [elementName];, then the element can be styled using the elementName.
(https://github.com/johanfrick/custom-sonos-card#css-styling) 

And for the buttons, here is an example:
```yaml
type: custom:sonos-player
skipAdditionalPlayerSwitches: true
styles:
  media-controls-icon:
    '--mdc-icon-size': 3rem
```
<img width="967" alt="image" src="https://user-images.githubusercontent.com/26582898/217084026-5a131014-e826-40a7-8a8d-64a6bcd7cd64.png">

**Comment by suneworld** [comment]
Thank you so much!
I've been trying to find the CSS for this but haven't had any progress.

Thank you, thank you, thank you!

-
Tryllekunstner Sune Alexandersen Narud
Øvre Måsan, Asker
Tlf 924 88 111
facebook.com/sunenarud
sunenarud.no <http://www.sunenarud.no>

**Reply by punxaphil** [reply]
I'm surprised you didn't figure it out, since you are a magician 😅

**Comment by suneworld** [comment]
Hahaha!
Thank you for the confidence :)

-
Tryllekunstner Sune Alexandersen Narud
Øvre Måsan, Asker
Tlf 924 88 111
facebook.com/sunenarud
sunenarud.no <http://www.sunenarud.no>

**Comment by beogaard** [comment]
Hej guys,
first let me thank you @johanfrick for yout phantastic work.
I'm struggeling with the same problem. I´m trying to enlarge the media buttons or a dashboard on a google hub.
But when i ad the snippet into my card nothing ever changes. Can you please push me in the right direction?

 ##
```
type: custom:sonos-card
sections:
  - player
  - volumes
  - groups
  - grouping
  - media browser
showVolumeUpAndDownButtons: true
mediaBrowserItemsPerRow: 4
heightPercentage: 105
widthPercentage: 180
topFavorites:
  - ''
mediaBrowserShowTitleForThumbnailIcons: true
hideGroupCurrentTrack: false
dynamicVolumeSlider: true
styles:
  media-controls-icon:
    '--mdc-icon-size': 3rem

```

**Reply by punxaphil** [reply, marked as answer]
This is not supported anymore. You need card mod for such hacks.

https://github.com/johanfrick/custom-sonos-card/discussions/194#discussioncomment-7542335

**Comment by punxaphil** [comment]
https://github.com/johanfrick/custom-sonos-card/discussions/104#discussioncomment-7659328

**Comment by beogaard** [comment]
thanks for the fast reply. with card mod it worked like a charm.
